apidoc/Titanium/UI/Clipboard/Clipboard.yml

```diff
@@ -151,7 +151,57 @@ methods:
         - type: Array<Dictionary>
     since: "5.5.0"
     osver: {ios: {min: "10.0"}}
+    platforms: [iphone, ipad]  
+
+  - name: remove
+    summary: Removes the clipboard.
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+
+  - name: name
+    summary: Create a new named clipboard.
+    description: Create a new clipboard with a given name. See the example on how to use this.
+    parameters:
+      - name: name
+        summary: name of clipboard.
+        type: String
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+    availability: creation
+
+  - name: unique
+    summary: Create a new clipboard identified by a unique system-generated name.
+    description: Create a new clipboard identified by a unique system-generated name. See the example on how to use this.
+    parameters:
+      - name: unique
+        summary: value of unique.
+        type: Boolean
+    since: "6.1.0"
     platforms: [iphone, ipad]
+    availability: creation
+
+  - name: persistent
+    summary: A Boolean value that indicates whether the clipboard is persistent.
+    description: When a clipboard is persistent, it continues to exist past app terminations and across system reboots. Clipboard that are not persistent only last until the owning (creating) app quits.
+    parameters:
+      - name: persistent
+        summary: value of persistent.
+        type: Boolean
+        default: false in case of named clipboard otherwise true.
+    since: "6.1.0"
+    osver: {ios: {max: "9.x.x"}}
+    platforms: [iphone, ipad]
+
+  - name: create
+    summary: Returns a clipboard identified by name, optionally creating it if it doesn,t exist.
+    description: If create is false, it will return the existing named clipboard but will not create new one. If create is true, it will return the existing clipboard (if exists) otherwise will create new clipboard with given name.
+    parameters:
+      - name: create
+        summary: value of create.
+        type: Boolean
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+    availability: creation
```

Comment:
What is this supposed to be? :-) Maybe we should rather call it different? Like `override` or `allowOverride`. And it doesn't return anything, so the docs should be updated to match the Apple docs. And check the line-break for `description`, you need to break it into multiple lines here.